### PR TITLE
Doc Updates QuickStart.md

### PR DESCRIPTION
**Pull request by gilliscraig**
Here's the updated QuickStart.md file based on http://sfdc.co/RefocusGuide. @meynet @iamigo

### CODE REVIEW

File: docs/QuickStart.md [hunk 1]
```diff
@@ -237,15 +276,8 @@ POST to ```v1/subjects/USA/child``` to create a
...
-Refocus will soon also allow you to use auth tokens to add an additional layer of security on the platform. Stay tuned!
-
-### Create your own lens
-//TODO point at ldk
-
-# Thank You!
-Congrats on the deployment of your Refocus! If you have any questions or feedback, please reach out to us.
+### Authorization Tokens Are Coming Soon!
 
+Soon you can use authorization tokens to add an additional layer of security on the platform. Stay tuned!
```
Comment (iamigo) [inline docs/QuickStart.md]: Might be better as:

Soon you can require that all API users use authorization tokens to add an additional layer of security on the platform.

File: docs/QuickStart.md [hunk 2]
```diff
@@ -1,64 +1,101 @@
...
+  Understand your data in new ways by gaining insight
+  
+- **Connect**  
+  Connect various monitoring tools together into one unified platform
+  
+- **Visualize**  
+  Visualize your system health in beautiful and unambiguous ways
+
+## Refocus Terminology
+Let’s familiarize you with the terms we use in Refocus.
```
Comment (iamigo) [inline docs/QuickStart.md]: Put this inline here rather than a link to a separate "terminology" document?

Comment (meynet) [inline docs/QuickStart.md]: If it is done this way we should use sub headers so we can url link to the sections

File: docs/QuickStart.md (hunk 1 above)
Comment (meynet) [inline docs/QuickStart.md]: I would rather leave this out of the quickstart. Security setup should be its own document or be part of the quickstart api calls

File: docs/QuickStart.md (hunk 2 above)
Comment (gilliscraig) [inline docs/QuickStart.md]: Having the terminology live here in this doc is the best way to keep it current. Also, I'll change the coding so the terms are heading, which can then be linked (like the TOC).

File: docs/QuickStart.md (hunk 1 above)
Comment (gilliscraig) [inline docs/QuickStart.md]: @iamigo I'll update the section with your changes and, @meynet I'll remove the authorization info from here.

File: docs/QuickStart.md [hunk 3]
```diff
@@ -1,64 +1,101 @@
...
-# Quick Start
+*OK, I've got Refocus deployed, and I'm signed in. Now what?*  
 
-There are two options to set up Refocus. Select the one that best meets your needs:
-- I want [one-click Heroku deployment](./QuickStartHeroku.md)
-- I want to [download and build and deploy locally](./QuickStartLocal.md)
+Think about how you want to structure your visualization based on the Refocus hierarchy. The relationship between subjects, aspects, and samples looks like this:
 
-Ensure that your deployment, either on Heroku or locally, has both Redis and PostgreSQL running. Once deployed, create an account and sign in.
+![Subject Hierarchy](
```
Comment (gilliscraig) [inline docs/QuickStart.md]: @iamigo @meynet I need permission to upload files to the repo folders so I can add images to this doc. This is a placeholder link to Jee's hierarchy graphic.

**Pull request merged**